src/buzzer.c

```diff
@@ -26,32 +26,34 @@ BuzzFreq_t requested_buzz_freq;
 uint8_t requested_buzz_volume;
 int32_t requested_buzz_length;
 
-// The buzzer is hooked up to PWM5 output, but contains an oscillator of its own so volume and freq are ignored for now. :(
+/*
+ * The buzzer is hooked up to PWM5 output, but contains an oscillator of
+ * its own so volume and freq are ignored for now. :(
```

Comment:
'For now', seems to be as long as no other hardware mods are added. Might be more clear to remove volume and freq from the code then?